src/ui/main.qml

```diff
@@ -339,6 +339,82 @@ Window {
         }
     }
 
+    Connections {
+        target: VPNErrorHandler
```

Comment:
We're just adding glean events into the android iap code. Can we add some glean events here too to see how often users get in different error states?

Reply:
Sure that is possible. I would include the glean events in `errorhandler.cpp` so that we create a glean event for every error that we have. Would that work?